pymemcache/client/base.py

```diff
@@ -823,20 +825,24 @@ def _store_cmd(self, name, values, expire, noreply, cas=None):
             self.close()
             raise
 
-    def _misc_cmd(self, cmd, cmd_name, noreply):
+    def _misc_cmd(self, cmds, cmd_name, noreply):
         if not self.sock:
             self._connect()
 
         try:
-            self.sock.sendall(cmd)
+            self.sock.sendall(b''.join(cmds))
```

Comment:
I have a slight preference for joining here with the `\r\n` instead of adding it to each `cmd`, but it's not a huge deal.